doc/source/user_guide/indexing.rst

```diff
@@ -573,7 +573,7 @@ without using a temporary variable.
 
 .. _indexing.deprecate_ix:
 
-IX indexer is deprecated
+Ix indexer is deprecated
```

Comment:
Hmm, since `ix` is / was more like a method this might be all lowercase "by definition" (just as you would never say something like Sort_values to refer to the `sort_values` method), so perhaps this might be one of the exceptions?

What do you think @datapythonista ?